refextract/references/kbs.py

```diff
@@ -42,6 +43,23 @@
 from ..documents.text import re_group_captured_multiple_space
 
 
+@contextlib.contextmanager
+def file_resolving(fpath, reader=None, **kwargs):
+    if isinstance(fpath, six.string_types):
+        print('Load kb path %s' % fpath)
```

Comment:
Yep I left them because I didn't know if they are needed! I think IOError message is quite descriptive when i/o exception is occured! So I will clean them ok?